task-sdk/src/airflow/sdk/definitions/baseoperator.py

```diff
@@ -1549,6 +1549,26 @@ def render_template_fields(
             jinja_env = self.get_template_env()
         self._do_render_template_fields(self, self.template_fields, context, jinja_env, set())
 
+    def pre_execute(self, context: Any):
+        """Execute right before self.execute() is called."""
```

Comment:
Add a pass to be explicit?

Reply:
`pass` is moot since we have docstrings. All the following are same :)


```python
def x(): ...

def x():
    pass

def x():
    """
```